scripts/cibuild/PublishArtifactsFromVsts.ps1

```diff
@@ -0,0 +1,67 @@
+<#
+.SYNOPSIS
+Sets build variables during a VSTS build dynamically.
+
+.DESCRIPTION
+This script is used to publish all offical nupkgs (from dev and release-* branches) to 
+myget feeds.
+
+#>
+
+param
+(
+    [Parameter(Mandatory=$True)]
+    [string]$NuGetBuildFeedUrl,
+
+    [Parameter(Mandatory=$True)]
+    [string]$NuGetBuildSymbolsFeedUrl,
+
+    [Parameter(Mandatory=$True)]
+    [string]$DotnetCoreFeedUrl,
+
+    [Parameter(Mandatory=$True)]
+    [string]$DotnetCoreSymbolsFeedUrl,
+
+    [Parameter(Mandatory=$True)]
+    [string]$NuGetBuildFeedApiKey,
+
+    [Parameter(Mandatory=$True)]
+    [string]$DotnetCoreFeedApiKey
+)
+
+function Push-ToMyGet {
+    [CmdletBinding(SupportsShouldProcess=$True)]
+    param(
+        [parameter(ValueFromPipeline=$True, Mandatory=$True, Position=0)]
+        [string[]] $NupkgFiles,
+        [string] $ApiKey,
+        [string] $BuildFeed,
+        [string] $SymbolSource
+    )
+    Process {
+        $NupkgFiles | %{
+            $Feed = Switch -Wildcard ($_) {
+                "*.symbols.nupkg" { $SymbolSource }
+                Default { $BuildFeed }
+            }
+            $opts = 'push', $_, $ApiKey, '-Source', $Feed
+            if ($VerbosePreference) {
+                $opts += '-verbosity', 'detailed'
+            }
+
+            if ($pscmdlet.ShouldProcess($_, "push to '${Feed}'")) {
+                Write-Output "$NuGetExe $opts"
+                & $NuGetExe $opts
+                if (-not $?) {
+                    Write-Error "Failed to push a package '$_' to myget feed '${Feed}'. Exit code: ${LASTEXITCODE}"
+                }
+            }
+        }
+    }
+}
+
+$NupkgsDir = Join-Path $env:BUILD_REPOSITORY_LOCALPATH artifacts nupkgs
+$NuGetExe = Join-Path $env:BUILD_REPOSITORY_LOCALPATH .nuget nuget.exe
+Write-Output "Publishing nupkgs from '$Nupkgs'"
+Get-Item "$NupkgsDir\*.nupkg" -Exclude '*Test*.nupkg', '*ProjectSystem.Interop*.nupkg' | Push-ToMyGet -ApiKey $NuGetBuildFeedApiKey -BuildFeed $NuGetBuildFeedUrl -SymbolSource $NuGetBuildSymbolsFeedUrl
+Get-Item "$NupkgsDir\NuGet.Build.Tasks.Pack*.nupkg" -Exclude '*Test*.nupkg', '*ProjectSystem.Interop*.nupkg' | Push-ToMyGet -ApiKey $DotnetCoreFeedApiKey -BuildFeed $DotnetCoreFeedUrl -SymbolSource $DotnetCoreSymbolsFeedUrl
```

Comment:
Do we still pack any test projects? I think since moving to cli 1.0.0 we should only be packing things that need to be pushed.